Using tags for billing (#8527)
DataDog/documentation: content/en/getting_started/tagging/_index.md

| [Logs][18]           | Filter logs search, analytics, patterns, live tail, and pipelines                                |
| [SLOs][19]           | Search for SLOs, grouped metric-based SLOs, grouped monitor-based SLOs                           |
| [Developers][20]     | Pull information or setup different areas in the UI with the API                                 |

> | [Billing][21]        | Report on Datadog usage by up to three tags of your choosing like `env`, `team`, `account_id`... |

Reviewer: ```suggestion
| [Billing][21]        | Report on Datadog usage by choosing up to three tags, for example: `env`, `team`, and `account_id` |
```
